NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -66,6 +68,17 @@ public int MaxConnections
             }
         }
 
+        public int InitialSyncTimeout
+        {
+            get => initialSyncTimeoutSetting;
+
+            set
+            {
+                Validate.IsTrue(value > 0, "InitialSyncTimeout must be greater than 0");
```

Comment:
I think we shouldn't be allowing small values (cause 0ms of timeout is quite strange like it is for 50, ...)